docker.d/configs/scriptworker.yaml

```diff
@@ -0,0 +1,56 @@
+provisioner_id: 'scriptworker-prov-v1'
+worker_group: 'treescriptworker-v1'
+worker_type:
+  $if: 'COT_PRODUCT == "thunderbird"'
+  then:
+    $if: 'ENV == "prod"'
+    then: 'treescript-dev'
+    else: 'treescript-v1'
+  else:
+    $if: 'ENV == "prod"'
+    then: 'treescript-dev'
+    else: 'treescript-v1'
```

Comment:
BTW, the names are the same for Firefox and Thunderbird. Is this intended?

Reply:
i think i forgot to save the file here

Reply:
Does Thunderbird have a treescript?